src/MongoDB/Manager.c

```diff
@@ -798,7 +798,13 @@ static void php_phongo_manager_free_object(zend_object* object) /* {{{ */
 	zend_object_std_dtor(&intern->std);
 
 	if (intern->client) {
-		MONGOC_DEBUG("Not destroying persistent client for Manager");
+		if (intern->use_persistent_client) {
+			MONGOC_DEBUG("Not destroying persistent client for Manager");
+		} else {
+			MONGOC_DEBUG("Destroying private client for Manager");
```

Comment:
Admittedly a nit because this is internal logging, but I'd suggest removing "private" and rephrasing as "Destroying client for Manager with disableClientPersistence=true" or "Destroying non-persistent client for Manager".